tests/e2e-image/dockerd-entrypoint.sh

```diff
@@ -35,7 +35,7 @@ fi
 if [ "$1" = 'dockerd' ]; then
 	# if we're running Docker, let's pipe through dind
 	# (and we'll run dind explicitly with "sh" since its shebang is /bin/bash)
-	set -- sh "$(which dind)" "$@" "--insecure-registry=$IP"
+	set -- sh "$(which dind)" "$@" "--insecure-registry=0.0.0.0/0"
```

Comment:
may I know why we need this change?  Just make sure this is not a shortcut which will cause problem in future...